airflow/contrib/operators/aws_athena_operator.py

```diff
@@ -47,7 +47,8 @@ class AWSAthenaOperator(BaseOperator):
 
     @apply_defaults
     def __init__(self, query, database, output_location, aws_conn_id='aws_default', client_request_token=None,
-                 query_execution_context=None, result_configuration=None, sleep_time=30, *args, **kwargs):
+                 query_execution_context=None, result_configuration=None, sleep_time=30, max_tries=None,
```

Comment:
You didn't update docstring for `max_tries`.

Reply:
Thx. I've updated it.